packages/contractkit/src/wrappers/Governance.ts

```diff
@@ -342,11 +342,12 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
   )
 
   /**
-   * Returns the proposal dequeue as list of proposal IDs.
+   * Returns the (existing) proposal dequeue as list of proposal IDs.
    */
-  getDequeue = proxyCall(this.contract.methods.getDequeue, undefined, (arrayObject) =>
-    arrayObject.map(valueToBigNumber)
-  )
+  async getDequeue() {
+    const dequeue = await this.contract.methods.getDequeue().call()
+    return dequeue.map(valueToBigNumber).filter((id) => !id.isZero())
```

Comment:
When would the IDs be zero?

Reply:
dequeued indices are reused after proposals are deleted (when expired)
the proposalID is set to zero in `deleteDequeuedProposal`, I will add a comment